### PR TITLE
修复: 按钮放大处理时显示有误的问题.

### PR DESCRIPTION
**Pull request by happyzhaole**

### CODE REVIEW

**Comment by dumganhar** [comment]
@zilongshanren , could you help to review this PR? Thanks.

File: extensions/CocoStudio/GUI/UIWidgets/UIButton.h [hunk 1]
```diff
@@ -171,6 +171,10 @@ class CC_EX_DLL Button : public Widget
     void setTitleFontName(const char* fontName);
     const char* getTitleFontName() const;
 
+	const std::string& getNormalFileName() const { return _normalFileName; }
```
Comment (zilongshanren) [inline extensions/CocoStudio/GUI/UIWidgets/UIButton.h:174]: The indentation is wrong here.

File: extensions/CocoStudio/GUI/UIWidgets/UIButton.cpp [hunk 2]
```diff
@@ -391,15 +391,8 @@ void Button::onPressStateChangedToNormal()
     }
     else
     {
-        if (_scale9Enabled)
```
Comment (zilongshanren) [inline extensions/CocoStudio/GUI/UIWidgets/UIButton.cpp:394]: This fix will break back compatibility. We can't just delete the `updateTextureRGBA` here.

File: extensions/CocoStudio/GUI/UIWidgets/UIButton.cpp [hunk 3]
```diff
@@ -424,15 +417,8 @@ void Button::onPressStateChangedToPressed()
         _buttonNormalRenderer->setVisible(true);
         _buttonClickedRenderer->setVisible(true);
         _buttonDisableRenderer->setVisible(false);
-        if (_scale9Enabled)
-        {
-            dynamic_cast<CCRGBAProtocol*>(_buttonNormalRenderer)->setColor(ccGRAY);
```
Comment (zilongshanren) [inline extensions/CocoStudio/GUI/UIWidgets/UIButton.cpp:429]: Why remove this line? It will break back compatibility.

**Comment by minggo** [comment]
@happyzhaole Thanks. But we can not accept Chinese in commit message. Could you please fix it?

**Pull request closed without merging**